cloud/deployment/inspect/inspect.go

```diff
@@ -372,3 +375,24 @@ func getTemplate(formattedDeployment *FormattedDeployment) FormattedDeployment {
 
 	return template
 }
+
+func getWorkloadIdentity(de *astro.Deployment) string {
+	// deployment workload identity only applies to AWS and GCP for now
+	fmt.Println(de.Cluster.CloudProvider)
```

Comment:
Do we need this print statement? or this was just for debugging?

Reply:
only for debugging